tools/stability-checker/internal/log/logfetcher.go

```diff
@@ -38,7 +40,7 @@ func NewPodLogFetcher(workingNamespace, podName string) (*PodLogFetcher, error)
 
 // GetLogsFromPod returns logs from pod
 func (p *PodLogFetcher) GetLogsFromPod() (io.ReadCloser, error) {
-	req := p.coreV1Client.Pods(p.workingNamespace).GetLogs(p.podName, &v1.PodLogOptions{})
+	req := p.coreV1Client.Pods(p.workingNamespace).GetLogs(p.podName, &v1.PodLogOptions{Container: stabilityCheckerContainerName})
```

Comment:
In `deploy.yaml` we should also have hardcoded name of the container, now it is the chart name:
```
name: {{ .Chart.Name }}
```